src/pages/overview/overviewPage.jsx

```diff
@@ -0,0 +1,22 @@
+import posts from "/src/constants/data.json"
+import {Link} from "react-router-dom";
+
+function Overview() {
+console.log(posts);
+
+    return (
+        <>
+        <h1>Overview page</h1>
+            <h4>Totaal aantal blogposts: {posts.length}</h4>
+            <ul>
+                {posts.map((post) => {
+                    return <li key={post.id}>
+                       <Link to="/blogposts/:blogId">{post.title}</Link> ({post.author}) {post.comments} reacties - {post.shares} keer gedeeld
```

Comment:
Doordat je in de Navigation aangeeft  dat je  een variabele gaat mee geven (:blogId) moet je in de url van de <Link> :blogId gaan vervangen voor het ID van de blog post.  je zou dit kunnen oplossen door  

```
<Link to={`/blogposts/${post.id}`}>{post.title}</Link>
```
 te gebruiken